src/transformers/models/chinese_clip/feature_extraction_chinese_clip.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for Chinese-CLIP."""
+
+from ...utils import logging
+from .image_processing_chinese_clip import ChineseCLIPImageProcessor
+
+
+logger = logging.get_logger(__name__)
+
+
+ChineseCLIPFeatureExtractor = ChineseCLIPImageProcessor
```

Comment:
For new vision models, I think we don't need to keep `FeatureExtractor`.
We have this for previous models as we want to keep backward compatibility.

Reply:
Feature extractors have not officially been deprecated, so they should be added until that point.

Reply:
Hi, this style of class organization is adapted from original CLIP, since we want to modify as less as possible to avoid more work on refactoring and testing. Currently these modules works in expectation. Is it acceptable anyway?